.github/workflows/dependency_review.yml

```diff
@@ -0,0 +1,31 @@
+# Dependency Review Action
+#
+# This Action will scan dependency manifest files that change as part of a Pull Request, surfacing known-vulnerable versions of the packages declared or updated in the PR. Once installed, if the workflow run is marked as required, PRs introducing known-vulnerable packages will be blocked from merging.
+#
+# Source repository: https://github.com/actions/dependency-review-action
+# Public documentation: https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-dependency-review#dependency-review-enforcement
+name: 'Dependency Review'
+on: 
+  push:
+    branches: 
+     - master
+     - production
+  pull_request:
+    branches: 
+     - master
+
+permissions:
+  actions: read
+  contents: read
+  pull-requests: write
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@v3
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@v3
```

Comment:
Please use [SHA hash](https://devopsjournal.io/blog/2021/12/11/GitHub-Actions-Maturity-Levels) instead of versions

v3.0.4 = f46c48ed6d4f1227fb2d9ea62bf6bcbed315589e

Reply:
Applied the changes